File: DatabaseMigrationSystem.Common/Dto/Request/MigrateTableRequest.cs
namespace DatabaseMigrationSystem.Common.Dto.Request;

public class MigrateTableRequest
{
    public string SourceTable { get; set; }
    
    public string SourceSchema { get; set; }
    
    public string DestinationTable { get; set; }
    
    public string DestinationSchema { get; set; }
    
    /// <summary>
    /// 
    /// </summary>
    public ColumnMapping[] ColumnsMapping { get; set; }
}

File: front/Database.Migration.System/src/app/api/models/migrate-table-request.ts
/* tslint:disable */
/* eslint-disable */
import { ColumnMapping } from './column-mapping';
export interface MigrateTableRequest {
  columnsMapping?: null | Array<ColumnMapping>;
  destinationSchema?: null | string;
  destinationTable?: null | string;
  sourceSchema?: null | string;
  sourceTable?: null | string;
}
